pyigm/guis/igmguesses.py

```diff
@@ -1510,11 +1522,13 @@ def reset(self):
         self.set_label()
 
     def update_component(self):
-        '''Values have changed'''
+        """Values have changed"""
+        self.update = False  # Avoids a bit of an internal loop
         self.Nwidget.set_text(self.component.attrib['logN'])
         self.zwidget.set_text(self.component.attrib['z'])
         self.bwidget.set_text(self.component.attrib['b'].value)
         self.Cwidget.set_text(self.component.comment)
+        self.update = True  # Avoids a bit of an internal loop
         if self.parent is not None:
```

Comment:
Trust me, I fussed with this quite a bit.

Things were crashing a lot previously..